hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -1292,61 +1292,36 @@ private class Emit[C](
         val shapet = emit(shapeIR)
         val rowMajort = emit(rowMajorIR)
 
-        val shapeAddress = mb.genFieldThisRef[Long]()
-
-        val oldShapeTuple = new CodePTuple(shapePType, shapeAddress)
-
-        val shapeVariables = (0 until nDims).map(_ => mb.newLocal[Long]()).toArray
-
-        def shapeBuilder(srvb: StagedRegionValueBuilder): Code[Unit] = {
-          Code(
-            srvb.start(),
-            Code.foreach(0 until nDims) { index =>
-              Code(
-                srvb.addLong(shapeVariables(index)),
-                srvb.advance())
-            })
-        }
-
-        val newResult = EmitCode.fromI(mb) { cb =>
-          cb.append(Code._println("Started making an ndarray"))
-
+        EmitCode.fromI(mb) { cb =>
           shapet.toI(cb).flatMap(cb) { case shapeTupleCode: PBaseStructCode =>
-            cb.append(Code._println("Flatmapped once"))
             datat.toI(cb).map(cb) { case dataCode: PIndexableCode =>
-              cb.append(Code._println("Mapped!"))
               val shapeTupleValue = shapeTupleCode.memoize(cb, "make_ndarray_shape")
               val dataValue = dataCode.memoize(cb, "make_ndarray_data")
               val dataPtr = dataValue.get.code.asInstanceOf[Code[Long]]
```

Comment:
We have a helper for this.
```suggestion
              val dataPtr = dataValue.get.tcode[Long]
```